status/format.go

```diff
@@ -196,3 +207,19 @@ func FormatFootnotes(taskRuns []*helpers.TaskRun) (string, error) {
 	}
 	return strings.Join(footnotes, "\n"), nil
 }
+
+// FormatPipelineURL accepts a name of application, pipelinerun, namespace and returns a complete pipelineURL.
+func FormatPipelineURL(pipelinerun string, namespace string) string {
+	console_url := os.Getenv("CONSOLE_URL")
+	if console_url == "" {
+		return "CONSOLE_URL_NOT_AVAILABLE"
+	}
+	buf := bytes.Buffer{}
+	ConsoleTemplate := os.Getenv("CONSOLE_URL")
```

Comment:
We already get the `CONSOLE_URL` from envvar on [line 213 above](https://github.com/redhat-appstudio/integration-service/pull/609/files#diff-e47e5b0d6143994b257b1b2d54a0525a6470ac5636ffebc2452517a7e367abbdR213), can we use that instead of calling it again?